panel/webserver_configuration.md

```diff
@@ -50,3 +50,48 @@ sudo ln -s /etc/apache2/sites-available/pterodactyl.conf /etc/apache2/sites-enab
 sudo a2enmod rewrite
 systemctl restart apache2
 ```
+
+## Caddy
+Caddy is a webserver with automatic https, this may come handy when you don't want to remake your ssl certificates every 90 days.
+::: danger
+Caddy is incompatible with other webservers, if you plan on running other websites on your panel you will have to add them to caddy too.
+:::
+/etc/caddy/Caddyfile:
+``` json
+panel.yourdomain.com {
+    # Set the directory root
+    root /var/www/pterodactyl/public
+    # Let caddy know where fastcgi is for php, edit "7.2" to your php-fpm version
+    fastcgi / /run/php/php7.2-fpm.sock php
+    # rewrite all requests to index.php, this is required for how pterodactyl works
+    rewrite {
+        to {path} {path}/ /index.php
+}
+```
+to turn it on:
+``` bash
+caddy -conf /etc/caddy/Caddyfile
+```
+To run caddy in the background use SystemD
+``` bash
+cat <<EOF > /etc/systemd/system/caddy@.service
```

Comment:
Is this actually the correct syntax for writing to the file? It seems incorrect.

Reply:
this was way you could just fill it into your bash shell, i've changed it to what should be in the file.